java/flight/flight-sql/src/test/java/org/apache/arrow/flight/sql/example/FlightSqlExample.java

```diff
@@ -253,36 +252,36 @@ public FlightSqlExample(final Location location, final String dbName) {
   }
 
   public static boolean removeDerbyDatabaseIfExists(final String dbName) {
-    boolean wasSuccess;
     final Path path = Paths.get("target" + File.separator + dbName);
 
     try (final Stream<Path> walk = Files.walk(path)) {
       /*
        * Iterate over all paths to delete, mapping each path to the outcome of its own
-       * deletion as a boolean representing whether or not each individual operation was
+       * deletion as a boolean representing whether each individual operation was
        * successful; then reduce all booleans into a single answer, and store that into
        * `wasSuccess`, which will later be returned by this method.
```

Comment:
wasSuccess doesn't exist anymore, let's update the comment

Reply:
Sorry missed that.